Projects/App/Sources/UI/Place/PlaceDetail/View/Views/PlaceInfoHeaderView.swift

```diff
@@ -0,0 +1,245 @@
+//
+//  PlaceInfoHeaderView.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/21.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import DesignSystem
+
+class PlaceInfoHeaderView: UITableViewHeaderFooterView {
+    
+    // MARK: - Properties
+    
+    private lazy var placeView = UIView()
+    private var kakaoUrl = ""
+    private var naverUrl = ""
+    
+    private lazy var categoryTagLabel: BasePaddingLabel = {
+        $0.font = .systemFont(ofSize: 11, weight: .bold)
+        $0.textColor = .white
+        $0.backgroundColor = .black
+        $0.layer.cornerRadius = 12
+        $0.layer.masksToBounds = true
+        return $0
+    }(BasePaddingLabel())
+    
+    private lazy var addReviewButton: UIButton = {
+        $0.setImage(UIImage(.btn_side_plus), for: .normal)
+        $0.addTarget(self, action: #selector(addReviewButtonDidTap), for: .touchUpInside)
+        return $0
+    }(UIButton(type: .custom))
+
+    private lazy var placeNameLabel: UILabel = {
+        $0.text = "(가게이름없음)"
+        $0.textColor = .black
+        $0.font = .systemFont(ofSize: 26, weight: .semibold)
+        return $0
+    }(UILabel())
+    
+    private var goodView = EvaluationItemView()
+    private var sosoView = EvaluationItemView()
+    private var badView = EvaluationItemView()
+    
+    private lazy var addressView: UIView = {
+        $0.backgroundColor = .zestyColor(.grayF6)
+        $0.layer.cornerRadius = 10
+        return $0
+    }(UIView())
+    
+    private lazy var addressLabel: UILabel = {
+        $0.text = "경북 포항시 남구 효자동 11길 24-1 1층 요기쿠시동"
+        $0.textColor = .zestyColor(.gray3C3C43)?.withAlphaComponent(0.6)
+        $0.font = .systemFont(ofSize: 11, weight: .regular)
+        $0.numberOfLines = 2
+        return $0
+    }(UILabel())
+    
+    private lazy var naverButton: UIButton = {
+        $0.setImage(UIImage(.btn_navermap), for: .normal)
+        $0.isUserInteractionEnabled = true
+        $0.addTarget(self, action: #selector(openNaverMap), for: .touchUpInside)
+        return $0
+    }(UIButton(type: .custom))
+    
+    private lazy var kakaoButton: UIButton = {
+        $0.setImage(UIImage(.btn_kakaomap), for: .normal)
+        $0.isUserInteractionEnabled = true
+        $0.addTarget(self, action: #selector(openKakaoMap), for: .touchUpInside)
+        return $0
+    }(UIButton(type: .custom))
+    
+    private lazy var reviewTitleLabel: UILabel = {
+        $0.text = "리뷰"
+        $0.textColor = .black
+        $0.font = .systemFont(ofSize: 26, weight: .bold)
+        return $0
+    }(UILabel())
+    
+    private lazy var lineView: UIView = {
+        $0.backgroundColor = .black
+        return $0
```

Comment:
p1;

<img width="513" alt="image" src="https://user-images.githubusercontent.com/75792767/197459341-206adc13-373c-4c53-b208-545579ae9e85.png">


```suggestion
        $0.backgroundColor = .black
        $0.layer.cornerRadius = 1
        $0.layer.masksToBounds = true
        return $0
```

살짝 라운드 있어서 적용 부탁요

Reply:
반영완료 2a49f6aeed5b9ccaeb26f5edb3e49f8ac2d8245f